normative/dtpr-stickers
# Refactors and redesigns place details page

# Notion card link(s)
- [Refactor Place Details](https://www.notion.so/normative/Refactor-Place-Detail-41f702892a664f5680d2ed8e6e19cb0f)

# Review APP
- [Heroku Review App](https://dtpr-sticker-chore-plac-olkkty.herokuapp.com/places/9UP3KDvEKqC-LvjYGibM6)

# PR Description (Proposed changes, approach, gotchas, etc)
- [x] Redesigns place page header
- [x] Adds function to group sensor by taxonomy props values 
- [x] Redesigns place page content 
- [x] Adds sorting by taxonomy 

# How to test it
- Run into your machine, navigate to `/pages/<sensorId>`
- Check if page is still working and designed as [Figma](https://www.figma.com/file/t1ZIjYmejw3bNc9hxMNzGB/Helpful-Places?node-id=188%3A5815)
